test/OrchardCore.Tests.Modules/ModuleSample/Controllers/ApiController.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OrchardCore.Queries;
+
+namespace ModuleSample.Controllers;
+
+
+[ApiController]
+public class ApiController : Controller
+{
+    [HttpPost, HttpGet]
+    [Route("~/moduleSample/tests/testCustomQuery")]
+    public async Task<IActionResult> TestCustomQueryAsync()
+    {
+        var result = new CustomQueryResult { Data = new { user = "Tom", age = 5 } };
```

Comment:
Hi @MikeAlhayek , I want to create an Api in this module to simulate the return result of a custom query, but I don't know why I can't access this endpoint in my tests and it keeps returning 404

Reply:
try removing `~/` from the route value.